OmiseSDK/PaymentCreatorController.swift

```diff
@@ -492,7 +492,8 @@ extension PaymentCreatorController {
         .installmentSCB,
         .promptPay,
         .trueMoney,
-        .pointsCiti
+        .pointsCiti,
+        .fpx
```

Comment:
FPX is for Malaysia right? So we should probably not add it to `thailandDefaultAvailableSourceMethods` instead we should create `malaysiaDefaultAvailableSourceMethods`?